src/ucp/tag/rndv.c

```diff
@@ -43,24 +44,37 @@ static void ucp_rndv_rma_request_send_buffer_dereg(ucp_request_t *sreq)
     }
 }
 
-size_t ucp_tag_rndv_pack_rkey(ucp_request_t *sreq, ucp_lane_index_t lane,
-                              void *rkey_buf, uint16_t *flags)
+size_t ucp_tag_rndv_pack_rkey(ucp_request_t *sreq, ucp_ep_h ep,
```

Comment:
sreq->req?  as this is not specific send anymore

Reply:
separate this function into two